extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -494,7 +494,8 @@ export class AmpIframe extends AMP.BaseElement {
   mutatedAttributesCallback(mutations) {
     const src = mutations['src'];
     if (src !== undefined) {
-      this.iframeSrc = this.transformSrc_(src);
+      this.iframeSrc =
+        /** @type {?string} */ (this.transformSrc_(src));
```

Comment:
Isn't this already the correct type?

Reply:
`transformSrc_` can return `undefined`. Without this change:
```js
extensions/amp-iframe/0.1/amp-iframe.js:497: ERROR - assignment to property iframeSrc of AmpIframe
found   : (string|undefined)
required: (null|string)
      this.iframeSrc =
      ^^^^^^^^^^^^^^^^
```